test/realtime/failure.test.js

```diff
@@ -185,6 +185,23 @@ define(['ably', 'shared_helper', 'async', 'chai'], function (Ably, helper, async
       }
     });
 
+    function checkIfRetryTimeoutsAreCorrect(retryTimeouts, calculationDelay = 0) {
+      function checkIsBetween(value, min, max) {
+        expect(value).to.be.above(min);
+        expect(value).to.be.below(max);
+      }
+
+      // Upper bound = min((retryAttempt + 2) / 3, 2) * initialTimeout
+      // Lower bound = 0.8 * Upper bound
+      checkIsBetween(retryTimeouts[0], 120, 150 + calculationDelay);
```

Comment:
I thought the whole point of this change was to make the tests work correctly even if we changed `disconnectedRetryTimeout` but you're still inlining timeout ranges here and elsewhere

Reply:
Actually, this makes tests easier to read. 
Also, it doesn't break the rule of value assertions. 
If we add formulas in the assertions, it adds the additional complexity of testing those formulas first. 
As far as I know, keeping formulas as assertions in unit tests are avoided.

Reply:
I have already added a comment denoting the same.

Reply:
Sorry but I honestly think this is less readable than before. The reason we wanted to change this test at all was because it included a magic number derived form the timeout configuration but this PR just adds even more of the same problem. Also, I don't think it changes the readability much, but extracting the assertions into functions just makes it harder to read the code from top to bottom so I preferred it before. IMO all we needed to do in the first place was just replace `50` with `disconnectedRetryTimeout / 3`

Reply:
I am not sure, what changes do you want me to make here?

Reply:
You should just make it so that we don't have inline numbers which make the test depend on a specific disconnectedRetryTimeout. I don't mind whether you refactor your existing changes or revert them and just change `50` to `disconnectedRetryTimeout / 3`